### DISCUSSION TITLE
No module named 'art'

### DISCUSSION THREAD

**Discussion by Sreyashi-Bhattacharjee** [Trusted-AI/adversarial-robustness-toolbox#1794] [Q&A]
I am not able to use art after installation. It shows "no module named art". 

_Installation code_: !pip install adversarial-robustness-toolbox[lightgbm]
_Python version_: 3.9.12
_ART version_: 1.11.0

_Steps followed_:
1. !pip install 
2. from art.estimators.classification import LightGBMClassifier

Please help. 

<img width="369" alt="art" src="https://user-images.githubusercontent.com/40518134/181172734-9ad1eb01-7fc5-410f-8f4a-52fee9f7dfa2.png">
<img width="518" alt="error" src="https://user-images.githubusercontent.com/40518134/181172742-a8a29c17-ede1-4350-a9b4-aab7e0687ed7.png">

**Comment by beat-buesser** [comment]
Hi @SBGitHub21 How did you install ART?

**Reply by Sreyashi-Bhattacharjee** [reply]
Hi @beat-buesser , I used -  !pip install adversarial-robustness-toolbox[lightgbm] on jupyter, as mentioned in the "Get started" page.

**Reply by beat-buesser** [reply]
@SBGitHub21 Did you restart the Jupyter kernel after installing?
